eventhubs/receive_eph.go

```diff
@@ -0,0 +1,114 @@
+package eventhubs
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	aad "github.com/Azure/azure-amqp-common-go/aad"
+	eventhubs "github.com/Azure/azure-event-hubs-go"
+	eph "github.com/Azure/azure-event-hubs-go/eph"
+	storageLeaser "github.com/Azure/azure-event-hubs-go/storage"
+	azure "github.com/Azure/go-autorest/autorest/azure"
+
+	// imports within this repo
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	storage "github.com/Azure-Samples/azure-sdk-for-go-samples/storage"
```

Comment:
[nit] For import paths where the last component is the same as what you wanted, I wouldn't create an alias. To me it actually gets in the way of identifying the conflict between `storage` in the samples repository and `storage` in the event hubs repository.